src/components/structures/RoomView.js

```diff
@@ -47,13 +47,12 @@ import UserProvider from '../../autocomplete/UserProvider';
 
 import RoomViewStore from '../../stores/RoomViewStore';
 
-var DEBUG = false;
+const DEBUG = false;
```

Comment:
I don't like this. If we attach a debugger we can't turn on `DEBUG` at run-time with this change.